sound/soc/sdca/sdca_device.c

```diff
@@ -22,3 +23,30 @@ void sdca_lookup_interface_revision(struct sdw_slave *slave)
 				 &slave->sdca_interface_revision);
 }
 EXPORT_SYMBOL_NS(sdca_lookup_interface_revision, SND_SOC_SDCA);
+
+bool sdca_device_quirk_match(struct sdw_slave *slave, enum sdca_quirk quirk)
```

Comment:
In my reading a 'quirk' does something active, while this function is more like match checking to sort codecs out from different parts or something?

If you would have `SDCA_QUIRKS_RT712_VA` then it would return true if the codec matches with that family.

I'm not sure if quirk is the right term for this.

Reply:
it's what we use for DMI quirks. If we have  a match, we do something.

Reply:
and 'quirk' is the term we use when we have hardware than requires additional information to be special-cased.